### PR TITLE
feat: About page

### LINKED ISSUE #70: Navigation bar text
The navigation menu text is "squeezed" to the top of the page. There should be more air between the text in the navigation manu and the edge of the page. See attatched photos below. 

The way it looks now: 

Screenshot 2021-03-17 16 01 51

The way the design looks: 

Screenshot 2021-03-17 16 01 45

### LINKED ISSUE #71: Navigation bar has to shrink when scrolling
The navigation bar will become slimmer when user is scrolling down. 

So from this: 
Screenshot 2021-03-17 16 57 29

To this: 

image-3

### LINKED ISSUE #94: Site margins
Please adjust margins. All content, including navigation, body and footer content should be aligned – not like this:
image

Footer background should also be full width.

### LINKED ISSUE #103: About page
Added images to the about page

### LINKED ISSUE #113: Header
On the "services", "about us" and "contact" pages the header loads in white with white letters. 

Screenshot 2021-03-25 08 44 12

After refreshing the letters turn black  with no background. 

Screenshot 2021-03-25 08 47 04

When scrolling the navigation bar should look like this: 

Screenshot 2021-03-25 09 15 29

### PR DESCRIPTION
**Pull request by dkostenko-bjerk**
Close #103
Close #113
Close #70 
Close #71
Close #94

### CODE REVIEW

File: web/src/pages/about.tsx [hunk 1]
```diff
@@ -1,12 +1,37 @@
 import { Box, Button, Grid, Heading, Label } from '@theme-ui/components';
 import { navigate } from 'gatsby';
-import Img from 'gatsby-image';
 import React from 'react';
 import { AboutList } from '../components/about-page/about-list';
 import { Container } from '../components/container';
 import { Layout } from '../components/layouts';
 
-// TODO: fix image showing when Sanity will be done
+const boxesData = [
```
Comment (cobraz) [inline web/src/pages/about.tsx:8]: I talk about this in #108 as well.

It is relevant here as well, IMHO. 

re https://github.com/bjerkio/website/pull/108#discussion_r600901491

**Review by cobraz — APPROVED** [review]
🙏

**Review by AnneMatilde — CHANGES_REQUESTED** [review]
Content looks good! The header background is missing on scroll. And the font is squeezed, but we have mentioned that in a different PR.

**Comment by AnnaBjerk** [comment]
@AnneMatilde 

> Content looks good! The header background is missing on scroll. And the font is squeezed, but we have mentioned that in a different PR.

Indeed, I made a new issue regarding header here:  #113  It pops up with wite letters and white background when opening the page and if the page is refreshed changes color to black. 

@dkostenko-bjerk

**Comment by dkostenko-bjerk** [comment]
Fixed bugs with the header

**Review by cobraz — APPROVED** [review]

**Review by AnnaBjerk — APPROVED** [review]
Looks good

**Comment by dkostenko-bjerk** [comment]
@AnneMatilde  Is #94 fixed here too?

**Review by AnneMatilde — CHANGES_REQUESTED** [review]
The about-page looks good! Please fix radius on button. 

This is how it looks now:
image


How it should look:
image

**Comment by dkostenko-bjerk** [comment]
@AnneMatilde , done

**Comment by AnneMatilde** [comment]
Sorry, #94 Looks good here @dkostenko-bjerk – except for the background on contact and footer.
Button is better, but seems like the font is wrong? 
image

**Comment by dkostenko-bjerk** [comment]
@AnneMatilde it's fixed

**Review by cobraz — APPROVED** [review]

**Review by AnneMatilde — APPROVED** [review]
Design looks good!

**Pull request merged**